automated-comments/go/raindrops/entry_signature_changed.md

```diff
@@ -0,0 +1,4 @@
+The signature of the `Convert` function should be `Convert(i int) string`.
```

Comment:
Should this perhaps be a general comment, where the function name and the expected signature are parameters? That way, this comment can be re-used in multiple analyzers.